UnityProject/Assets/Scripts/Players/PlayerManager.cs

```diff
@@ -66,6 +66,8 @@ public bool CreatePlayer(PlayerDTO playerDTO)
 
             activePlayers.Add(PlayerId(playerDTO.id), player);
 
+			//TODO: Random texture?
```

Comment:
Create issue on github and you can remove this comment. That code won't go there anyway most likely